### ISSUE TITLE
Improvements to `select_input_styling` app

### ISSUE THREAD

**Issue by davidruvolo51** [davidruvolo51/shinyAppTutorials#2]
The event listeners for `onfocus`, `onblur` and `onchange` should be defined in a js file as opposed to inline declaration. Accessibility concerns will also need to be addressed. Future changes should documented and turned into a new post.

**Comment by davidruvolo51** [comment]
Event listeners are now in `www/js/index.js`. Formal write up in progress
